tests/base/src/test/java/org/wildfly/security/auth/client/CredentialStoreSaslAuthenticationTest.java

```diff
@@ -19,6 +19,7 @@
 package org.wildfly.security.auth.client;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
```

Comment:
Just a very minor thing, the imports should be in alphabetical order so `fail` should be under `assertTrue`.